app/components/settings/email-preferences-section.js

```diff
@@ -3,4 +3,9 @@ import Ember from 'ember';
 const { Component } = Ember;
 
 export default Component.extend({
+  actions: {
+    save(notificationType, setting) {
```

Comment:
In email-preferences-section.hbs you can directly use `{{action savePreference}}` or `(action savePreference)` right ? Why do you need to go with a round about way ?